customer-service/automatic-record-creation/RemoveUnrefrencedQueues/README.md

```diff
@@ -0,0 +1,55 @@
+---
+languages:
+- csharp
+products:
+- dynamics-customer-service
+page_type: sample
+level: beginner
+role: developer
+description: "Sample that shows how Automatic Record Creation for Dynamics 365 Customer Service can be customized to remove unreferenced queues from received emails"
+---
+
+# Sample: ARC Multi-Related - Remove Unrefrenced Queues
+
+This sample shows how to write a simple plug-in that updates the activity parties on an email removing any related item's created by ARC that are no longer party to the email.  This assumes that the org has enabled ARC's Multi-Related for Email feature.
+
+## Use cases Covered
+
+1. An external user sends an email to a queue monitored by Automatic Record Creation (ARC).  ARC creates a case for that email and adds the created case to the email's related activity party list. Because of this relationship the email is shown on the case's timeline view.  The user then forwards an email from that chain to another mailbox (for example, the agent's manager).  This custimization allows for organizations that  do not want that forwarded email on the timeline like it would be by default.
+2. Similarlly, If a user replies to an email that was sent to multiple ARC Rules (for example, email sent to support and engineering queues) and the user removes one of those mailboxes from a reply, this customization would remove the cases not assosciated with a recipient. For example, if sent to only support queue, any cases created from the engineering queue would be removed.
+
+## How to run this sample
+
+1. Download or clone the [Dynamics 365 Samples](https://github.com/microsoft/Dynamics365-Apps-Samples) repo so that you have a local copy. This sample is located under customer-service\automatic-record-creation\RemoveUnrefrencedQueues.
+2. Open the sample solution in Visual Studio, navigate to the project's properties, and verify the assembly will be signed during the build. Press F6 to build the sample's assembly (RemoveUnrefrencedQueues.dll).
+3. Run the Plug-in Registration tool and register the sample's assembly in the D365 server's sandbox and database. When registering a plug-in step, specify the "Create" message, email table, and syncronous mode on the Post Create stage.
+4. Create two or more Automatic Record Creation rules with unique queue mailboxes.
+5. Trigger case creation of the ARC rule by sending an email to the configured queue for one of the ARC rules.
+6. Test the plugin by replying or forwarding that email to different ARC enabled queues.
+6. When you are done testing, unregister the plugin assembly and step.
```

Comment:
In markdown, you don't need to number items. You should just always use `1. `, and the rendering will number it for you.